src/popup/popupHeaderFooter.js

```
@@ -0,0 +1,129 @@
+        });
+    } catch (err) {
+        logDebug("Error loading header file:", err);
+    }
+}
+
+// Unload Footer handler
+unloadFooterBtn.onclick = function () {
+    logDebug("Unload Footer button clicked");
+    chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+        chrome.tabs.sendMessage(tabs[0].id, { greeting: 'unloadFooter' }, function (response) {
+            if (response && response.code) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)